draft-wang-ppm-differential-privacy.md

```diff
@@ -456,9 +473,74 @@ class DpMechanism:
 
 > TODO: Specify a Gaussian sampler
 
-## Randomized Response {#rr}
+## Binary Randomized Response {#rr}
+
+The binary randomized response mechanism specified in Appendix C.1 of
+{{MJTB+22}} is `EPSILON_0`-DP at the Client level, in the deletion-DP model
+(Definition II.4 of {{EFMR+20}}).
```

Comment:
(Follow up on https://github.com/wangshan/draft-wang-ppm-differential-privacy/pull/22#discussion_r1364371532) This looks fine except "Client level" is not defined. What about "in the OC trust model"?

Also, we should define deletion-DP informally here so that the reader doesn't need to look this up.

```suggestion
The binary randomized response mechanism specified in Appendix C.1 of
{{MJTB+22}} is `EPSILON_0`-DP at the OC trust model, in the deletion-DP model
(Definition II.4 of {{EFMR+20}}).
```